### DISCUSSION TITLE
New lab idea Ansible

### DISCUSSION THREAD

**Discussion by davep75** [rhel-labs/instruqt#528] [Ideas]
How about a lab on creating an Ansible role?  One that is complex enough to use variables, a handler, a template and a file.

**Comment by myee111** [comment]
Hey @davep75, thanks for the suggestion. I think labs that are Ansible related are best left to our Ansible team who already make Instruqt labs. I'll let them know about your request.

**Comment by leogallego** [comment]
HI @davep75, Leo here from the Ansible team. We are working on a lab that will include this topic using the new Ansible development tools package and VS Code for creating collections. Stay tuned!
